### PR TITLE
DPI-950 update OTA documentation to v2.1

### PR DESCRIPTION
**Pull request by oysteinsigholt**
align content with `Ruter DPI - OTA Messages 2.1.pdf`

### CODE REVIEW

File: docs/ota.md [hunk 1]
```diff
@@ -63,7 +73,7 @@ Notify PTA BO that the bus is starting a block.
 | Name             | Type    | Description                   |
 |------------------|---------|-------------------------------|
 | eventTimestamp   | string  | ISO 8601, UTC                 |
-| vehicleNumber    | integer | same as for the bridged topic |
+| vehicleNumber    | integer | Same as the &lt;vehicleid&gt; in the topic, a VIN |
```
Comment (eivindga) [inline docs/ota.md]: I don't think this is correct any more. It is not an integer, but a string. 

In general we should request @andrelin or someone else from sanntid to review the documentation for sign-on and sign-off, as they have better insights into this format than me.

Comment (phototropic) [reply docs/ota.md]: Version 2.1 pdf has string as the type so it’s just an omission that should be corrected.

Comment (oysteinsigholt) [reply docs/ota.md]: @phototropic is correct that this was an oversight by me. It has been corrected.

File: docs/ota.md [hunk 2]
```diff
@@ -51,10 +61,10 @@ Notify PTA BO that the bus is starting a block.
...
 ```json
 {
-  "eventTimestamp": "2017-10-31T12:45:50Z",
-  "vehicleNumber": "12345",
-  "blockId": "1234:34",
-  "vehicleJourneyId": "35:ABC"
+   "eventTimestamp": "2019-05-04T15:03:10Z",
+   "vehicleNumber": "E2345678901234567",
+   "blockId": "3103",
+   "vehicleJourneyId": "294"
```
Comment (andrelin) [inline docs/ota.md]: Her aksepterer vi også IDen som blockID-Starttid
Der starttid må vere ISO 8601 date i lokal tidssone, altså CE(S)T.
Det burde nok også vere dokumentert

Comment (oysteinsigholt) [reply docs/ota.md]: Takk for input!

**Pull request merged**